apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/BindingTests.cs

```diff
@@ -240,22 +242,25 @@ public async Task BindNumericEmptyArray()
             new DateTime?[] { });
 
         [SkippableFact]
+        [Trait(Constants.SkipOnEmulator, Constants.Yes)]
```

Comment:
With the trait in place, can we remove the Skip.If calls? (It feels like a recipe for inconsistency, requiring both.)